packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -372,6 +374,15 @@ export function createTextInstance(
   return textNode;
 }
 
+export function getCurrentEventPriority(): EventPriority {
+  const currentEvent = window.event;
+  if (currentEvent === undefined) {
+    return DefaultEvent;
+  }
+  // TODO: Is this fast enough? It reads from a Map.
+  return getEventPriorityForPluginSystem(currentEvent.type);
```

Comment:
Yeah it should be a giant switch

Reply:
there you go

![21350DD3-D479-4033-AA6E-B6B34466B846](https://user-images.githubusercontent.com/810438/107104757-f263d780-681a-11eb-8d85-1bcd497e04b2.jpeg)